hips/draw/tests/test_simple.py

```diff
@@ -40,17 +40,17 @@
 ]
 
 
-@remote_data
-@pytest.mark.parametrize('pars', make_sky_image_pars)
-def test_make_sky_image(pars):
-    hips_survey = HipsSurveyProperties.fetch(url=pars['url'])
-    geometry = make_test_wcs_geometry(case=2)
-    image = make_sky_image(geometry=geometry, hips_survey=hips_survey, tile_format=pars['file_format'])
-    assert image.shape == pars['shape']
-    assert image.dtype == pars['dtype']
-    assert_allclose(np.sum(image), pars['data_sum'])
-    assert_allclose(image[200, 994], pars['data_1'])
-    assert_allclose(image[200, 995], pars['data_2'])
+# @remote_data
+# @pytest.mark.parametrize('pars', make_sky_image_pars)
+# def test_make_sky_image(pars):
+#     hips_survey = HipsSurveyProperties.fetch(url=pars['url'])
+#     geometry = make_test_wcs_geometry(case=2)
+#     image = make_sky_image(geometry=geometry, hips_survey=hips_survey, tile_format=pars['file_format'])
+#     assert image.shape == pars['shape']
+#     assert image.dtype == pars['dtype']
+#     assert_allclose(np.sum(image), pars['data_sum'])
+#     assert_allclose(image[200, 994], pars['data_1'])
+#     assert_allclose(image[200, 995], pars['data_2'])
```

Comment:
Remove this change where you commented out code.
If you want to just run certain tests, use `pytest` to select which ones.